index.html

```diff
@@ -54,7 +54,7 @@ <h3 class="text-animation"> I'm here to help with <span></span></h3>
             <div class="btn-group">
                 <a href="#" class="btn">Hire</a>
                 <a href="#contact" class="btn">Contact</a>
-                <a href="#" class="bot" id="toggleBotButton">Open Bot</a>
+                <a href="#" class="bot" id="toggleBotButton">Bot</a>
```

Comment:
I see the changes, I think we will have to reconvene on this one. How will a user know if it is closed or open?